### PR TITLE
[2.0.0-rc02] bump android and ios to 2.0.0-rc02

### PR DESCRIPTION
**Pull request by nwadams**
release rc02 with latest fixes for android and ios

### CODE REVIEW

**Review by benbeltzer — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -25,8 +25,11 @@
     "lint": "eslint \"**/*.{js,ts,tsx}\"",
     "release": "release-it",
     "example": "yarn --cwd example",
+    "example2": "yarn --cwd example2",
```
Comment (benbeltzer) [inline package.json:28]: please delete this, plus `pods2` and `bootstrap2`

Comment (nwadams) [reply package.json]: done

File: package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
     "release": "release-it",
     "example": "yarn --cwd example",
     "pods": "cd example && pod-install --quiet",
-    "bootstrap": "yarn example && yarn && yarn pods"
+    "bootstrap": "yarn example && yarn && yarn pods",
```
Comment (benbeltzer) [inline package.json:29]: complains about trailing comma

**Review by benbeltzer — APPROVED** [review]
approved pending one comma

**Pull request merged**